SPECS/kernel/config

```diff
@@ -2957,7 +2957,7 @@ CONFIG_IPMI_SI=m
 # CONFIG_IPMI_SSIF is not set
```

Comment:
kernel.spec (and associated `kernel-signed-x64`/`kernel-signed-aarch64`) release number should be updated too.

Reply:
Looks like I need to update them all, regardless, I messed up my rebase an lost my spec file change logs as well.